doris/install.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+wget --timestamping https://apache-doris-releases.oss-accelerate.aliyuncs.com/${DORIS_PACKAGE}.tar.gz
```

Comment:
Sorry, we will need to use the official installation steps instead of installing some binary from some random-looking source. Please follow the installation instructions in `Use Docker for Quick Deployment` at https://doris.apache.org/docs/3.0/gettingStarted/quick-start. (I actually tried to follow the steps but got weird errors and the MySQL client could not connect to the cluster).

(Side note: Further down on the above linked page, it says "Operating System: It is recommended to use AMD/ARM mainstream Linux environments such as CentOS 7.1 or Ubuntu 16.04 and above." Both operating systems are extremely outdated by now, suggest to update the recommendation.)

Reply:
`https://apache-doris-releases.oss-accelerate.aliyuncs.com/${DORIS_PACKAGE}.tar.gz` is regular download source from https://doris.apache.org/download, there were some errors in the previous doris documentation and are being fixed